v1.1/licensing.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Licensing
+summary: Request and set trial and enterprise license keys for CockroachDB
+toc: false
+---
+
+This page shows you how to request and set the trial and enterprise license keys for CockroachDB.
+
+<div id="toc"></div>
+
+CockroachDB distributes a single binary that contains both core and [enterprise features](https://www.cockroachlabs.com/pricing/). You can use core features without any license key. However, to use the enterprise features, you need either a trial or an enterprise license key.
+
+## Types of Licenses
+
+You can request a trial license or an enterprise license:
+
+- Trial license: A trial license enables you to try out CockroachDB enterprise features for 30 days. After 30 days, you can upgrade to the enterprise license by [requesting](licensing.html#request-a-trial-or-an-enterprise-license-key) and [setting](licensing.html#set-the-trial-or-enterprise-license-key) an enterprise license key.
+- Enterprise license: An enterprise license enables you to use CockroachDB enterprise features for longer periods (one year or more).
+
+## Request a Trial or an Enterprise License Key
+
+[Contact us](https://www.cockroachlabs.com/pricing/sales/) to request a trial or an enterprise license key.
+
+## Set the Trial or Enterprise License Key
+
+As the CockroachDB `root` user, open the [built-in SQL shell](use-the-built-in-sql-client.html) in the insecure or secure mode, as per your CockroachDB setup. In the following example, we assume that CockroachDB is running in the insecure mode. Then use the `SET CLUSTER SETTING` command to set the name of your organization and the license key:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql --insecure
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+>  SET CLUSTER SETTING cluster.organization = 'Acme Company'; 
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+>  SET CLUSTER SETTING enterprise.license = 'xxxxxxxxxxxx’;
+~~~
+
+## Verify the License Key is Set
```

Comment:
nit: I don't think `is Set` is necessary in the header.

Reply:
Fixed.